endpoints/info/bidders.go

```diff
@@ -35,11 +35,19 @@ func NewBiddersEndpoint(aliases map[string]string) httprouter.Handle {
 	})
 }
 
-// NewBiddersEndpoint implements /info/bidders/*
+// NewBidderDetailsEndpoint implements /info/bidders/*
 func NewBidderDetailsEndpoint(infos adapters.BidderInfos, aliases map[string]string) httprouter.Handle {
+	// Build a new map that's basically a copy of "infos" but will also contain
+	// alias bidder infos
+	var allBidderInfo = make(map[string]adapters.BidderInfo, len(infos)+len(aliases))
+
 	// Build all the responses up front, since there are a finite number and it won't use much memory.
 	responses := make(map[string]json.RawMessage, len(infos))
```

Comment:
We should do the fix above to add `+len(aliases` to the `responses` size.

Reply:
Ah! Good catch. Thanks! I will update it